app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/Appbar.kt

```diff
@@ -26,7 +26,7 @@ fun Appbar(
         ) {
             Icon(
                 painter = painterResource(id = R.drawable.ic_back),
-                tint = AppTheme.colors.coinInfoAppBarIconTint,
+                tint = co.nimblehq.compose.crypto.core.theme.AppTheme.colors.coinInfoAppBarIconTint,
```

Comment:
Can we refactor this full path in this PR? There is a lot. 🙇 
```suggestion
                tint = AppTheme.colors.coinInfoAppBarIconTint,
```